cimgui_structs_accessor.cpp

```diff
@@ -1,3 +1,5 @@
+// Code generated by cmd/codegen of https://github.com/AllenDang/cimgui-go DO NOT EDIT!
```

Comment:
Would be better if it looked like this:

```go
// Code generated by cmd/codegen from https://github.com/AllenDang/cimgui-go.
// DO NOT EDIT.
```

Reply:
good idea - the line might be too long now